README.md

```diff
@@ -136,22 +136,22 @@ The continued development and maintenance of GitHub1s is made possible by these
 
 - [febaoshan/edge-extensions-github-code-viewer](https://github.com/febaoshan/edge-extensions-github-code-viewer) - [Chrome Web Store](https://chrome.google.com/webstore/detail/github-code-viewer/ecddapgifccgblebfibdgkagfbdagjfn)
 
-- [Darkempire78/Github1s-Extension](https://github.com/Darkempire78/Github1s-Extension)
-- [zvizvi/Github Web IDE](https://github.com/zvizvi/github-web-ide) - [Chrome Web Store](https://chrome.google.com/webstore/detail/adjiklnjodbiaioggfpbpkhbfcnhgkfe)
+- [Darkempire78/GitHub1s-Extension](https://github.com/Darkempire78/GitHub1s-Extension)
+- [zvizvi/GitHub Web IDE](https://github.com/zvizvi/github-web-ide) - [Chrome Web Store](https://chrome.google.com/webstore/detail/adjiklnjodbiaioggfpbpkhbfcnhgkfe)
```

Comment:
Similarly, @zvizvi can safely adjust the title of their tool (although sometimes using someone else's brand is problematic)...